<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>771</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Thẻ tháng</string>
     </attribute>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>521</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Mã thẻ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Trạng thái</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Biển số xe</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Thẻ lượt</string>
     </attribute>
     <widget class="QTableWidget" name="table2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>521</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Mã thẻ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Trạng thái</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Biển số xe</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>55</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Xóa thẻ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImport">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>55</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Thêm nhiều thẻ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>55</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Thêm thẻ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Trạng thái:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnCard">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Mã thẻ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Biển số:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnPlate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="cbbStatus">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Tất cả</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chưa sử dụng</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Đã sử dụng</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>650</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
